thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/alert/DetectionAlertTaskFactory.java

```diff
@@ -103,8 +100,8 @@ public DetectionAlertFilter loadAlertFilter(DetectionAlertConfigDTO alertConfig,
       Preconditions.checkNotNull(alertSchemes.get(alertSchemeType));
       Preconditions.checkNotNull(alertSchemes.get(alertSchemeType).get(PROP_CLASS_NAME));
       Constructor<?> constructor = Class.forName(alertSchemes.get(alertSchemeType).get(PROP_CLASS_NAME).toString().trim())
-          .getConstructor(ADContentFormatterContext.class, ThirdEyeAnomalyConfiguration.class, DetectionAlertFilterResult.class);
-      detectionAlertSchemeSet.add((DetectionAlertScheme) constructor.newInstance(adContext, thirdeyeConfig, result));
+          .getConstructor(DetectionAlertConfigDTO.class, ThirdEyeAnomalyConfiguration.class, DetectionAlertFilterResult.class);
```

Comment:
Will change the function signature break tests in "third eye-external"?

Reply:
yes, I will raise a RB an thirdeye-external as well.